src/sentry_core.c

```diff
@@ -71,9 +71,20 @@ sentry_init(sentry_options_t *options)
         SENTRY_DEBUG("falling back to non-absolute database path");
         options->database_path = database_path;
     }
-    SENTRY_DEBUGF("Using database path \"%" SENTRY_PATH_PRI "\"",
+    SENTRY_DEBUGF("using database path \"%" SENTRY_PATH_PRI "\"",
         options->database_path->path);
 
+    sentry_path_t *handler_path = options->handler_path;
+    if (handler_path) {
+        options->handler_path = sentry__path_absolute(handler_path);
+        if (options->handler_path) {
+            sentry__path_free(handler_path);
+        } else {
+            SENTRY_DEBUG("falling back to non-absolute handler path");
+            options->handler_path = handler_path;
```

Comment:
This fallback will cause the `execvp` crash. I believe we should make sure that the crashpad backend logs an error and skips initialization in this case. This would be another argument for moving this block into the backend.